src/Metadata/Resource/ResourceMetadata.php

```diff
@@ -25,15 +25,17 @@
     private $iri;
     private $itemOperations;
     private $collectionOperations;
+    private $graphqlQuery;
     private $attributes;
 
-    public function __construct(string $shortName = null, string $description = null, string $iri = null, array $itemOperations = null, array $collectionOperations = null, array $attributes = null)
+    public function __construct(string $shortName = null, string $description = null, string $iri = null, array $itemOperations = null, array $collectionOperations = null, array $attributes = null, array $graphqlQuery = null)
     {
         $this->shortName = $shortName;
         $this->description = $description;
         $this->iri = $iri;
         $this->itemOperations = $itemOperations;
         $this->collectionOperations = $collectionOperations;
+        $this->graphqlQuery = $graphqlQuery;
```

Comment:
That disturbs me to have references to GraphQL here when it's supposed to be a bridge.